### DISCUSSION TITLE
HAPPINESS-BUFF complejidad

### DISCUSSION THREAD

**Discussion by mfriasi** [IIC2133-PUC/2024-2#95] [Tarea 1] [answered]
En la parte de happiness-buff, dice que debemos aumentar la felicidad del pinguino con id especificado sin superar la complejidad O(n+log(s)). Mi pregunta es si la operación que no debe superar esa complejidad es la de aumentar la felicidad del pinguino o también incluye el reordenar el heap?

**Comment by tomastrivino** [comment, marked as answer]
La complejidad solicitada es la de todo el evento `HAPPINESS-BUFF`. Esto incluye el aumentar la felicidad y todo lo demás :)
